wagtail/utils/decorators.py

```diff
@@ -57,3 +57,26 @@ def cache_clear(self):
         """Clear the cached value."""
         # Named after lru_cache.cache_clear
         self.cache.pop(self.cls, None)
+
+
+def xframe_options_sameorigin(view_func):
```

Comment:
I opted for the rename instead of adding an `override=True` because I think the latter would require quite a bit of changes to the decorator’s implementation to allow it to be used with and without arguments? (e.g. `@xframe_options_sameorigin` and `@xframe_options_sameorigin(override=False)`)